src/lsm/table_memory.zig

```diff
@@ -295,11 +299,19 @@ pub fn TableMemoryType(comptime Table: type) type {
             return key_from_value(&a) < key_from_value(&b);
         }
 
+        pub fn key_range_contains(table: *const TableMemory, key: Key) bool {
+            assert(table.value_context.sorted);
+
+            if (table.count() == 0) return false;
+            if (key < table.key_min() or key > table.key_max()) return false;
+            return true;
```

Comment:
Maybe `return table.key_min() <= key and key <= table.key_max()`? The "ternary comparison" like `low <= x and x <= hi` is a bit easier to read, usually.